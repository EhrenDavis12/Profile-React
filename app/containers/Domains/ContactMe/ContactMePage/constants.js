/*
 *
 * ContactMePage constants
 *
 */

export const SUBMIT_MESSAGE_FORM = 'app/ContactMePage/SUBMIT_MESSAGE_FORM';
export const SUBMIT_MESSAGE_FORM_SUCCEEDED =
  'app/ContactMePage/SUBMIT_MESSAGE_FORM_SUCCEEDED';
export const SUBMIT_MESSAGE_FORM_FAILED =
  'app/ContactMePage/SUBMIT_MESSAGE_FORM_FAILED';
