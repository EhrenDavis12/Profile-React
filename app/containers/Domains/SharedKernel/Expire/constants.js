/*
 *
 * Expire constants
 *
 */

export const CLEAR_TIMER_ACTION = 'app/Expire/CLEAR_TIMER_ACTION';
export const SET_TIMER_ACTION = 'app/Expire/SET_TIMER_ACTION';
export const SHOW_CHILDREN_ACTION = 'app/Expire/SHOW_CHILDREN';
export const HIDE_CHILDREN_ACTION = 'app/Expire/HIDE_CHILDREN';
export const SET_TIMER_SUCCEEDED = 'app/Expire/SET_TIMER_SUCCEEDED';
export const CLEAR_TIMER_SUCCEEDED = 'app/Expire/CLEAR_TIMER_SUCCEEDED';
