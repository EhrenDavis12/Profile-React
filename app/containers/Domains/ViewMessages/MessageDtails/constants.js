/*
 *
 * MessageDtails constants
 *
 */

export const DEFAULT_ACTION = 'app/MessageDtails/DEFAULT_ACTION';
