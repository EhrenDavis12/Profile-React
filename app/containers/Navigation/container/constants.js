/*
 *
 * Navigation constants
 *
 */

export const DEFAULT_ACTION = 'app/Navigation/DEFAULT_ACTION';
export const TOGGLE_DRAWER = 'app/Navigation/TOGGLE_DRAWER';
export const SELECT_LINK = 'app/Navigation/SELECT_LINK';
