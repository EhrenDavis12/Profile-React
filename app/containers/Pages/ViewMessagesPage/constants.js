/*
 *
 * ViewMessagesPage constants
 *
 */

export const REQUEST_USER = 'app/ViewMessagesPage/REQUEST_USER';
export const REQUEST_USER_SUCCEEDED =
  'app/ViewMessagesPage/REQUEST_USER_SUCCEEDED';
export const REQUEST_USER_FAILED = 'app/ViewMessagesPage/REQUEST_USER_FAILED';

export const REQUEST_USER_MESSAGES =
  'app/ViewMessagesPage/REQUEST_USER_MESSAGES';
export const REQUEST_USER_MESSAGES_SUCCEEDED =
  'app/ViewMessagesPage/REQUEST_USER_MESSAGES_SUCCEEDED';
export const REQUEST_USER_MESSAGES_FAILED =
  'app/ViewMessagesPage/REQUEST_USER_MESSAGES_FAILED';
