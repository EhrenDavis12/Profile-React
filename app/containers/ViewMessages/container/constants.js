/*
 *
 * ViewMessagesPage constants
 *
 */

export const REQUEST_USER = 'app/ViewMessagesPage/REQUEST_USER';
export const REQUEST_USER_SUCCEEDED =
  'app/ViewMessagesPage/REQUEST_USER_SUCCEEDED';
export const REQUEST_USER_FAILED = 'app/ViewMessagesPage/REQUEST_USER_FAILED';

export const REQUEST_USER_MESSAGES =
  'app/ViewMessagesPage/REQUEST_USER_MESSAGES';
export const REQUEST_USER_MESSAGES_SUCCEEDED =
  'app/ViewMessagesPage/REQUEST_USER_MESSAGES_SUCCEEDED';
export const REQUEST_USER_MESSAGES_FAILED =
  'app/ViewMessagesPage/REQUEST_USER_MESSAGES_FAILED';

export const SELECT_USER_MESSAGE = 'app/ViewMessagesPage/SELECT_USER_MESSAGE';
export const SELECT_USER_MESSAGE_SUCCEEDED =
  'app/ViewMessagesPage/SELECT_USER_MESSAGE_SUCCEEDED';
export const SELECT_USER_MESSAGE_FAILED =
  'app/ViewMessagesPage/SELECT_USER_MESSAGE_FAILED';
