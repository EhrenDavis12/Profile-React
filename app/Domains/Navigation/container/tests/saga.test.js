/**
 * Test sagas
 */

/* eslint-disable redux-saga/yield-effects */
import { takeLatest, put, all } from 'redux-saga/effects';

import { push } from 'react-router-redux';
import { SELECT_LINK } from '../constants';
import rootSaga, { pushLinkSaga, pushLinkStart } from '../saga';

const action = { selectedLink: { LinkTo: 'testSagasLink' } };

describe('navigationSaga Saga', () => {
  /* it('should execute the pushLinkSaga', () => {
    const generator = pushLinkSaga();
    const putDescriptor = generator.next(pushLinkStart).value;
    expect(putDescriptor).toEqual(takeLatest(SELECT_LINK, pushLinkStart));
    expect(putDescriptor).toMatchSnapshot();
  });

  it('should execute the pushLinkStart', () => {
    const generator = pushLinkStart(action);
    const putDescriptor = generator.next().value;
    expect(putDescriptor).toEqual(put(push(action.selectedLink.LinkTo)));
    expect(putDescriptor).toMatchSnapshot();
  });

  it('should execute the rootSaga', () => {
    const generator = rootSaga();
    const putDescriptor = generator.next().value;
    expect(putDescriptor).toEqual(all([pushLinkSaga()]));
    expect(putDescriptor).toMatchSnapshot();
  }); */

  it('should execute the blank rootSaga', () => {
    const generator = rootSaga();
    const putDescriptor = generator.next().value;
    expect(putDescriptor).toEqual(all());
    expect(putDescriptor).toMatchSnapshot();
  });
});
