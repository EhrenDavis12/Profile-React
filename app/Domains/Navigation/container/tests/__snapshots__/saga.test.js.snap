// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`navigationSaga Saga should execute the pushLinkSaga 1`] = `
Object {
  "@@redux-saga/IO": true,
  "FORK": Object {
    "args": Array [
      "app/Navigation/SELECT_LINK",
      [Function],
    ],
    "context": null,
    "fn": [Function],
  },
}
`;

exports[`navigationSaga Saga should execute the pushLinkStart 1`] = `
Object {
  "@@redux-saga/IO": true,
  "PUT": Object {
    "action": Object {
      "payload": Object {
        "args": Array [
          "testSagasLink",
        ],
        "method": "push",
      },
      "type": "@@router/CALL_HISTORY_METHOD",
    },
    "channel": null,
  },
}
`;

exports[`navigationSaga Saga should execute the rootSaga 1`] = `
Object {
  "@@redux-saga/IO": true,
  "ALL": Array [
    GeneratorFunctionPrototype {
      "_invoke": [Function],
    },
  ],
}
`;
